docs/query-builder.txt

```diff
@@ -9,588 +9,1095 @@ Query Builder
+   :end-before: end aggregation avg
+
+.. _laravel-query-builder-aggregation-sum:
+
+Summed Value of a Field Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``sum()``
+query builder method to return the numerical total of
+the ``imdb.votes`` values from the entire ``movies``
+collection:
+
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation sum
+   :end-before: end aggregation sum
+
+.. _laravel-query-builder-aggregate-matched:
+
+Aggregate Matched Results Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``sum()``
+query builder method to return the numerical total of
+the ``imdb.votes`` values from movies released in
+after the year "2000":
 
-Selected columns that are not grouped will be aggregated with the ``$last``
-function.
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation with filter
+   :end-before: end aggregation with filter
 
-.. code-block:: php
+.. _laravel-modify-results-query-builder:
 
-   $users =
-       Users::groupBy('title')
-           ->get(['title', 'name']);
+Modify Query Results
+--------------------
 
-**Distinct**
+This section includes query builder examples for the
+following functions that modify the order and format
+of query results:
+
+- :ref:`Order results by the value of a field <laravel-query-builder-orderby>`
+- :ref:`Omit a specified number of results <laravel-query-builder-skip>`
+- :ref:`Show a subset of fields and array values in the results <laravel-query-builder-project>`
+- :ref:`Paginate the results <laravel-query-builder-paginate>`
+
+.. _laravel-query-builder-orderby:
+
+Order Results Example
+~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to use the ``orderBy()``
+query builder method to arrange the results that match
+the filter specified in the ``title`` field by the
+``imdb.rating`` value in descending order:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/query-builder/MovieController.php
+      :language: php
+      :dedent:
+      :start-after: begin query orderBy
+      :end-before: end query orderBy
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+        { "title": "Back to the Future", "imdb": { "rating":8.5,"votes":636511,"id":88763 }, ... },
+        { "title": "Back to the Future Part II", "imdb": { "rating":7.8,"votes":292539,"id":96874 }, ... },
+        { "title": "Back to the Future Part III", "imdb": {"rating":7.4,"votes":242390,"id":99088 }, ... },
+        ...
+      ]
+
+.. _laravel-query-builder-skip:
+
+Omit a Specified Number of Results Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to use the ``skip()`` query builder method to
+omit the first "4" results that match the filter specified in the ``title``
```

Comment:
```suggestion
omit the first 4 results that match the filter specified in the ``title``
```

Reply:
Thanks for the suggestion. I'll use "four" as the quantifier since it's less than 10.